cmd/benchdb/main.go

```
@@ -291,6 +293,15 @@ func (ut *benchDB) manualGC(done chan bool) {
 	}
 }
 
+func (ut *benchDB) query(spec string) {
+	strs := strings.Split(spec, ":")
+	sql := strs[0]
+	count, _ := strconv.Atoi(strs[1])
```

Why discard this error?